src/protocol/mod.rs

```diff
@@ -32,9 +32,9 @@ use std::{fmt::Debug, hash::Hash};
 pub trait Step: AsRef<str> {}
 
 // In test code, allow a string (or string reference) to be used as a `Step`.
-#[cfg(debug_assertions)]
+#[cfg(any(feature = "test-fixture", debug_assertions))]
 impl Step for String {}
-#[cfg(debug_assertions)]
+#[cfg(any(feature = "test-fixture", debug_assertions))]
 impl Step for str {}
```

Comment:
Oh, of course.

With this, you might want to remove [this hack](https://github.com/martinthomson/raw-ipa/blob/9fe8280313335c64cc2ead19cad171e96ccc0033/src/test_fixture/circuit.rs#L32-L65).

Reply:
ah good catch. removed it but certainly don't want to use strings long term.